media/redesign/stylus/components/wiki/revision-list.styl

```diff
@@ -97,3 +97,7 @@ html[dir=rtl] .revision-list-controls:before {
     width: 460px;
 }
 
+/* make the Compare selected revisions button align to layout */
+.revision-list-controls .fixed {
+    bidi-style(left, 24px, right, 24px);
```

Comment:
That makes sense now that you've explained it, hopefully I can remember for future commits :)